packages/gp2-components/src/molecules/FilterPill.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  borderWidth,
+  crossSmallIcon,
+  pixels,
+} from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import { ReactNode } from 'react';
+import colors from '../templates/colors';
+
+const { rem } = pixels;
+
+const pillStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  width: 'fit-content',
```

Comment:
I don't think it needs a lot of this css code, the height and width are gonna be set by the content + padding, we don't need to justify the content and I'm not sure we need the align either

Reply:
The width : 'fit-content' is necessary, because it will cause the pill to expand to the other side of the page, if not applied. The align is also necessary because it will move away from the center. After testing a little bit more, the one I can remove is the justifyContent.